### PR TITLE
Always sign URLs

### PR DESCRIPTION
**Pull request by martin-ro**
- sign all url's, private and public to ensure private files are displayed in curator
- truncate the ulr for nicer display

### CODE REVIEW

**Comment by awcodes** [comment]
I'm gonna have to do some testing with this. I'm not confident that always returning a signed temp url is the best case. I'm also not sure at this time if that'll have an effect on the curations or not.

**Comment by martin-ro** [comment]
One thing that could break: I think local disk drivers don't support `temporaryUrl` so at least a check for the driver is needed.

File: src/Models/Media.php
Comment (awcodes) [inline src/Models/Media.php:1]: Can you try it with this:
```php
protected function url(): Attribute
{
    return Attribute::make(
        get: function () {
            if (Storage::disk($this->disk)->getVisibility($this->path) === 'private') {
                return Storage::disk($this->disk)->temporaryUrl(
                    $this->path,
                    now()->addMinutes(5)
                );
            }

            return Storage::disk($this->disk)->url($this->path);
        },
    );
}
```

Comment (martin-ro) [reply src/Models/Media.php:1]: Tested with `public` & `private` and both works as expected.

**Pull request merged**